applications/fishing-map/src/features/map/map.draw.utils.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  Dataset,
+  DatasetCategory,
+  DatasetConfiguration,
+  DatasetTypes,
+} from '@globalfishingwatch/api-types'
+import { DrawFeature, DrawPointPosition } from './MapDraw'
+
+export const getDrawDatasetDefinition = (name: string): Partial<Dataset> => {
+  return {
+    name,
+    type: DatasetTypes.Context,
+    category: DatasetCategory.Context,
+    configuration: {
+      format: 'geojson',
+      geometryType: 'draw',
+    } as DatasetConfiguration,
+  }
+}
+
+export const getFileWithFeatures = (name: string, features: DrawFeature[]) => {
+  return new File(
+    [
+      JSON.stringify({
+        type: 'FeatureCollection',
+        features: features,
+      }),
+    ],
+    `${name}.json`,
+    {
+      type: 'application/json',
+    }
+  )
+}
+
+export const getFeaturesPrecisionRounded = (features: DrawFeature[]): DrawFeature[] => {
+  return features.map(
+    (feature) =>
+      ({
+        ...feature,
+        geometry: {
+          ...feature.geometry,
+          coordinates: feature.geometry.coordinates.map((coordinate) =>
+            coordinate.map((points) => points.map((point) => Math.round(point * 100000) / 100000))
```

Comment:
It would be nice to try to enforce a standard re this. Maybe start by moving this to a util + a config const?